docs/development.md

```diff
@@ -100,18 +106,18 @@ To debug the code generator:
 
 ### Built in commands
 
-There are a number of custom rush commands to help with development. See the `.scripts` folder for more.
+There are a number of custom pnpm commands to help with development. See the `.scripts` folder for more.
 
 To run `go build` and `go vet` on every generated module:
 
 ```terminal
-rush buildvet
+pnpm buildvet
```

Comment:
These scripts are no longer global like they were with rushjs. So, if you execute them from some place other than the root of the repo you have to add the `-r` switch so `pnpm` will recursively look for them. Probably good to call this out above (maybe in the sentence on line 109).